src/components/SupportContainer.tsx

```diff
@@ -0,0 +1,100 @@
+// /src/components/SupportContainer.tsx
+
+// React modules
+import React, { useEffect, useState } from "react";
+// array of expert data
+import expertsData from "../../static/data/community-experts.json";
+
+// Component to render community experts and their metadata
+const SupportContainer = () => {
+  // State management
+  const [experts, setExperts] = useState([]);
```

Comment:
I Should have picked up earlier that we should have typed this, as it would save us some problems downstream, e.g.:

```typescript
const [experts, setExperts] = useState<IExpert[]>([]);
```

Reply:
Ah yes, I can see how this would have been missed in the switch from js to ts.

I understand and agree